pallets/loans/src/lib.rs

```diff
@@ -1015,10 +1016,10 @@ impl<T: Config> Pallet<T> {
             .checked_sub(repay_amount)
             .ok_or(ArithmeticError::Underflow)?;
         let total_borrows = Self::total_borrows(currency_id);
-        let total_borrows_new = total_borrows
-            .checked_sub(repay_amount)
-            .ok_or(ArithmeticError::Underflow)?;
+        // TODO : total_borrows should contain interests
+        let total_borrows_new = total_borrows.saturating_sub(repay_amount);
```

Comment:
total_borrows < repay_amount and caused this issue:)

we need to find a way to make total_borrows contain also interests